content/docs/reference-react-dom-server.md

```diff
@@ -10,116 +10,120 @@ The `ReactDOMServer` object enables you to render components to static markup. T
 
 ```js
 // ES modules
-import ReactDOMServer from 'react-dom/server';
+import * as ReactDOMServer from 'react-dom/server';
 // CommonJS
 var ReactDOMServer = require('react-dom/server');
 ```
 
 ## Overview {#overview}
 
-The following methods can be used in both the server and browser environments:
-
-- [`renderToString()`](#rendertostring)
-- [`renderToStaticMarkup()`](#rendertostaticmarkup)
-
-These additional methods depend on a package (`stream`) that is **only available on the server**, and won't work in the browser.
+These methods are only available in the **environments with [Node.js Streams](https://nodejs.dev/learn/nodejs-streams):**
 
 - [`renderToPipeableStream()`](#rendertopipeablestream)
-- [`renderToReadableStream()`](#rendertoreadablestream)
 - [`renderToNodeStream()`](#rendertonodestream) (Deprecated)
 - [`renderToStaticNodeStream()`](#rendertostaticnodestream)
 
-* * *
-
-## Reference {#reference}
-
-### `renderToString()` {#rendertostring}
-
-```javascript
-ReactDOMServer.renderToString(element)
-```
-
-Render a React element to its initial HTML. React will return an HTML string. You can use this method to generate HTML on the server and send the markup down on the initial request for faster page loads and to allow search engines to crawl your pages for SEO purposes.
-
-If you call [`ReactDOM.hydrateRoot()`](/docs/react-dom-client.html#hydrateroot) on a node that already has this server-rendered markup, React will preserve it and only attach event handlers, allowing you to have a very performant first-load experience.
-
-* * *
-
-### `renderToStaticMarkup()` {#rendertostaticmarkup}
+These methods are only available in the **environments with [Web Streams](https://developer.mozilla.org/en-US/docs/Web/API/Streams_API)** (this includes browsers, Deno, and some modern edge runtimes):
 
-```javascript
-ReactDOMServer.renderToStaticMarkup(element)
-```
+- [`renderToReadableStream()`](#rendertoreadablestream)
 
-Similar to [`renderToString`](#rendertostring), except this doesn't create extra DOM attributes that React uses internally, such as `data-reactroot`. This is useful if you want to use React as a simple static page generator, as stripping away the extra attributes can save some bytes.
+The following methods can be used in both the server and Web API environments:
```

Comment:
I guess for these two it's actually for any *other* JS environment that doesn't support any of the above. Their existence is basically just because they don't have any complicated dependencies.

Reply:
https://github.com/reactjs/reactjs.org/blob/main/content/docs/reference-react-dom-server.md

Reply:
https://github.com/reactjs/reactjs.org/commit/1015a73e1abe829e61612474485549e034596df8